client/src/views/Cart.vue

```diff
@@ -71,7 +71,7 @@ export default {
   },
   computed: {
     rows() {
-      return this.$store.state.cart;
+      return this.$store.getters.cart;
```

Comment:
I guess moving the cart into a store module changed the structure of the state object. So getters should be used instead to decouple a bit from how the state modules are organized.